docfx_project/ziti/clients/tunneler-overview.md

```diff
@@ -1,4 +1,7 @@
-A tunneler is purpose-built software designed to connect applications which are not Ziti-aware to the Ziti Network.
-Ziti provides tunnelers for each major operating system.
+Ziti Tunnellers allow pre-existing, unmodified TCP/IP applications to access or provide services on a secure Ziti network. A Ziti Tunneler is a Ziti-native application that communicates with local peers using TCP/IP and proxies the payload to/from a Ziti service.
+
+Note that embedding the Ziti SDK directly into a standalone application or plugin is preferable to using a Ziti Tunneler, if possible. Such a Ziti-native application is secured and encrypted to the farthest edges of the communication path - all the way to the application's internal buffers. A Ziti Tunneler cannot secure the communication between itself and the TCP/IP application.

```

Comment:
"Note that embedding the Ziti SDK directly into a standalone application or plugin is preferable to using a Ziti Tunneler, if possible" this is not relevant to this page imo and should be removed. 

I like the whole "application embedded is superior" vibe but I don't like the delivery. I think this nuance is probably not useful on this page unless it comes along with the "ZTNA/ZTHA/ZTAA" diagram. That's probably where it belongs anyway imo - not here.  A new page talking about the three "levels" of zero trust is where it belongs